integration_tests/projects/008_pure_python_models/models/staging/model_c.py

```diff
@@ -1,29 +1,15 @@
 """
-DEPENDENCY: ref("model_b")
+DEPENDENCY:
+# not really used, but we put it to make sure it is picked up
 - ref('model_a')
 """
-import os
-from functools import reduce
-from datetime import datetime
+from _fal_testing.utils import create_dynamic_artifact
+from fal.typing import *
 
-# does not get picked up, so we put it in the docstring
-df = ref(*["model_b"])
+df = ref("model_b")
 
 df["my_bool"] = True
 
-# BigQuery only:
-# df["my_ts"] = datetime(2022, 6, 27, 17, 12, 25)
-
 write_to_model(df)
 
-model_name = context.current_model.name
-
-output = f"Model name: {model_name}"
-output = output + f"\nStatus: {context.current_model.status}"
-
-output = output + "\nModel dataframe name: {model_name}"
-temp_dir = os.environ["temp_dir"]
-
-write_dir = open(reduce(os.path.join, [temp_dir, model_name + ".txt"]), "w")
-write_dir.write(output)
-write_dir.close()
+create_dynamic_artifact(context)
```

Comment:
AFAIK `create_dynamic_artifact` only creates artifacts for scripts (e.g., `model.script_name.txt` instead of `model.txt`), is this intended?

Reply:
Thanks for pointing it out!